src/org/whstsa/library/gui/api/LibraryManagerUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.whstsa.library.gui.api;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.whstsa.library.api.IPerson;
+import org.whstsa.library.api.ObservableReference;
+import org.whstsa.library.api.books.IBook;
+import org.whstsa.library.api.library.ILibrary;
+import org.whstsa.library.api.library.IMember;
+import org.whstsa.library.db.ObjectDelegate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LibraryManagerUtils {
+
+    public static List<IPerson> returnPeople(ObservableReference<ILibrary> libraryReference) {
+        List<IPerson> people = new ArrayList<>();
+        libraryReference.poll().getMembers().forEach(member -> people.add(member.getPerson()));
+
+        return people;
+    }
+
+    public static ObservableList<String>  getMemberNames(ObservableReference<ILibrary> libraryReference) {
+        List<String> list = new ArrayList<>();
+        for (int i = 0; i < libraryReference.poll().getMembers().size(); i++) {
```

Comment:
I would advise using `.forEach` or even `for(T : List<T>` when possible. Only use traditional iteration when you need to use the index.